backend/src/main/resources/migration/utf/mdb_columns.csv

```diff
@@ -0,0 +1,214 @@
+database_name,TABLE_NAME,COLUMN_NAME,DATA_TYPE,maximum_length
```

Comment:
It looks like this file is generated.  Let's not store it in git since we can recreate it later.